Dockerfile

```diff
@@ -7,14 +7,14 @@ FROM $LITELLM_BASE_IMAGE
 # Set the working directory to /app
 WORKDIR /app
 
-# Copy the current directory contents into the container at /app
-COPY . /app
+COPY requirements.txt .
 
-# Install any needed packages specified in requirements.txt
 RUN pip wheel --no-cache-dir --wheel-dir=wheels -r requirements.txt
 RUN pip install --no-cache-dir --find-links=wheels -r requirements.txt
 
+COPY . /app
+
 EXPOSE 4000/tcp
 
 # Start the litellm proxy, using the `litellm` cli command https://docs.litellm.ai/docs/simple_proxy
-ENTRYPOINT litellm --config /app/proxy_server_config.yaml --port 4000
+CMD ["litellm", "--config", "/app/proxy_server_config.yaml", "--host", "0.0.0.0", "--port", "4000"]
```

Comment:
FYI I've made updates to the litellm dockerfile, it should always include `ENTRYPOINT` so users can pass litellm args